custom_model_runner/datarobot_drum/drum/language_predictors/java_predictor/pom.xml

```diff
@@ -41,6 +41,23 @@
             <artifactId>opencsv</artifactId>
             <version>5.2</version>
         </dependency>
+       <!-- https://mvnrepository.com/artifact/ai.h2o/h2o-genmodel -->
+        <dependency>
+            <groupId>ai.h2o</groupId>
+            <artifactId>h2o-genmodel</artifactId>
+            <version>3.30.0.6</version>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/ai.h2o/h2o-genmodel-ext-xgboost -->
+        <dependency>
+            <groupId>ai.h2o</groupId>
+            <artifactId>h2o-genmodel-ext-xgboost</artifactId>
+            <version>3.30.0.6</version>
+        </dependency>
+        <dependency>
+            <groupId>org.scala-lang</groupId>
+            <artifactId>scala-library</artifactId>
+            <version>2.12.8</version>
+        </dependency>
```

Comment:
Does these deps come with every drum install? Is it big?

Reply:
custom scoring code jar goes from 3.4MB to right around 9.3MB

Reply:
drum is ~9 MB now.